src/vcSettings.h

```diff
@@ -123,7 +123,16 @@ struct vcVisualizationSettings
   int minIntensity;
   int maxIntensity;
 
-  udFloat2 displacement;
+  struct
+  {
+    float x;
+    float y;
```

Comment:
This should be `udFloat2` not individual components, these two variables need to be contiguous and using a `udFloat2` ensures that.